bpdb/__init__.py

```diff
@@ -106,9 +106,7 @@ def main():
             print("\t" + " ".join(sys.argv[1:]))
         except SystemExit:
             # In most cases SystemExit does not warrant a post-mortem session.
-            print(
-                "The program exited via sys.exit(). Exit status: ",
-            )
+            print("The program exited via sys.exit(). Exit status: ",)
```

Comment:
These formatting changes aren't desired, hopefully running black will fix them? Maybe you have a different autoformatter or different settings?